### DISCUSSION TITLE
Where is CONNECTOR_ID ?

### DISCUSSION THREAD

**Discussion by 0xbbuddha** [misje/opencti-wazuh-connector#92] [Q&A] [answered]
Hello, I am currently trying out the Wazuh/OpenCTI integration. The only problem is that I can't find the CONNECTOR_ID parameter, which is the last parameter I need to launch the connector-wazuh. Thank you in advance!

**Comment by misje** [comment, marked as answer]
It is mentioned under [required settings](https://misje.github.io/opencti-wazuh-connector/required_settings.html). This is just an ID that OpenCTI uses to distinguish connector,s and it can probably be anything. However, the OpenCTI docs suggest to use something like an UUID.

**Comment by 0xbbuddha** [comment]
So, can I use a uuid generator ?

**Reply by misje** [reply]
Yes, any UUID works.
